src/js/port_handler.js

```diff
@@ -170,6 +170,11 @@ PortHandler.detectPort = function(currentPorts) {
             }
         });
 
+        // Signal board verification
+        if (GUI.active_tab === 'firmware_flasher') {
+            TABS.firmware_flasher.verifiedBoard = 'verifyBoard';
```

Comment:
Re-using this isn't very clean - better introduce an extra boolean value like `boardNeedsVerification`.

Reply:
Fixed!